### PR TITLE
backtest : auto sync

### PR DESCRIPTION
**Pull request by tony1223**

### CODE REVIEW

File: pkg/cmd/backtest.go [hunk 1]
```diff
@@ -194,71 +194,50 @@ var BacktestCmd = &cobra.Command{
...
+						if err := backtestService.SyncExist(ctx, sourceExchange, symbol, syncFromTime, time.Now(), interval); err != nil {
+							return err
 						}
-
-						if err := <-errC; err != nil {
+						//return fmt.Errorf("the sync-from-time you gave %s is earlier than the previous sync-start-time %s. "+
+						//	"re-syncing data from the earlier date before your first sync is not support,"+
+						//	"please clean up the kline table and restart a new sync",
+						//	syncFromTime,
+						//	firstKLine.EndTime)
```
Comment (c9s) [inline pkg/cmd/backtest.go:227]: remove unused code?

File: pkg/service/backtest.go [hunk 2]
```diff
@@ -22,15 +23,15 @@ type BacktestService struct {
...
+	//}
 
-	if lastKLine != nil {
-		log.Infof("found the last %s kline data checkpoint %s", symbol, lastKLine.EndTime)
-		startTime = lastKLine.StartTime.Add(time.Minute)
-	}
+	//if lastKLine != nil {
+	//	log.Infof("found the last %s kline data checkpoint %s", symbol, lastKLine.EndTime)
+	//	startTime = lastKLine.StartTime.Add(time.Minute)
+	//}
```
Comment (c9s) [inline pkg/service/backtest.go:34]: remove unused code?

**Review by c9s — APPROVED** [review]

File: pkg/service/backtest.go (hunk 2 above)
Comment (tony1223) [reply pkg/service/backtest.go]: ok

File: pkg/cmd/backtest.go (hunk 1 above)
Comment (tony1223) [reply pkg/cmd/backtest.go]: ok

**Pull request merged**